### ISSUE TITLE
Apple domains blocked

### ISSUE THREAD

**Issue by cborchichi** [AdAway/adaway.github.io#191]
**Describe the bug**
http://adaway.org/hosts.txt includes subdomains of apple.com that are necessary for apple devices to run, for example iadsdk.apple.com

**To Reproduce**
1. search for iadsdk.apple.com

**Expected behavior**
Eliminate apple.com subdomains from the list.

**Screenshots**

**Device**
 - AdAway version: n/a
 - Device: pfSense
 - Android version: n/a
 - Root version: n/a

**Additional context**
Add any other context like logcat about the problem here. Use adb logcat if you have developer settings enabled on your device or use any application like [MatLog](https://play.google.com/store/apps/details?id=com.pluscubed.matlog) to save logs. Include the crash report in the issue in a code block, a file attachment, or put it in a gist and provide link to that gist.

**Comment by spirillen** [comment]
I happens to block a lot of these inside my RPZ, and persoanlly I haven't experienced any issues on my phone, so could you please elaborate your specific issue?

# Rpz records
|id|domain_id|name|type|content|
|--|---------|----|----|-------|
|1715066|36|iadsdk.apple.com.edgekey.net.rpz.mypdns.cloud|CNAME|.|
|3283414|36|iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283415|36|ca.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283416|36|cf.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283417|36|tr.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283418|36|cs.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283419|36|news.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283420|36|ut.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283421|36|su.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|
|3283422|36|www.iadsdk.apple.com.rpz.mypdns.cloud|CNAME|.|

Hosts records
https://github.com/AdAway/adaway.github.io/blob/9d8ad3d3135815f6ec6169179754429bb7565f4a/hosts.txt#L2364-L2370

**Comment by jawz101** [comment]
are you sure they're necessary for Apple apps to run?  As I understand, iad usually stands for intertstitial ads (that is, banner ads within apps.)  I'll remove them because I don't use Apple but is it really causing problems?

I guess Apple so iAd is just iWhatever they brand.  Still.  Ad

https://developer.apple.com/support/iad/
https://old.reddit.com/r/pihole/comments/dk3sv9/apple_news_ads/
https://en.wikipedia.org/wiki/IAd

**Comment by llacb47** [comment]
> includes subdomains of apple.com that are necessary for apple devices to run

Not true, I encounter no issues

**Comment by cborchichi** [comment]
I had issues with trustd daemon in MacOS (Catalina); but if is for common good leave it as is. I whitelisted the ones that I need. IOS seems to have no issues.

**Comment by jawz101** [comment]
Well if it breaks the desktop I'd prefer to remove them.  Which ones were the problem?

**Comment by jawz101** [comment]
nm.  I went ahead and removed them.

**Comment by spirillen** [comment]
I would be reconsidering this one.

First of there is a bell ringing in the back of my head, that this only applies to Catalina, Still looking for confirmation.

Next the first search I found lead me to this post: <https://macintouch.com/community/index.php?threads/apple-security-and-privacy.710/page-43#post-31274>

Next searches comes up with one more: <https://www.snbforums.com/threads/ad-blocking.20992/>

Unfortunately the search result on the particular domain is rather limited, but I think the trail seems like good pointers

As we already know, this domain solely is used for adware as part of the iAd Framework: <https://developer.apple.com/documentation/iad> I would recommend to start with to ask people to whitelist it, until we might have collected more information on what it breaks, how common is it. So for once llacb47 and I have found something to agree on :smiley: 

Also the user-account : `cborchichi` is 2 days old and the holder can very well be part of the iAd developer team.
